docs/cep/sink/sink-types/inMemory.md

```diff
@@ -2,20 +2,20 @@
 title: inMemory (Sink)
```

Comment:
@diana-macrometa - Let's remove this `inMemory` source/sink from the official docs. A malicious customer could crash the `CEP` overloading it.